newIDE/app/src/GameDashboard/GameRegistration.js

```diff
@@ -152,9 +152,11 @@ export const GameRegistration = ({
 
   React.useEffect(
     () => {
-      loadGame();
+      if (!game) {
+        loadGame();
+      }
     },
-    [loadGame]
+    [loadGame, game]
```

Comment:
@ClementPasteau Interesting, why was this necessary?

Reply:
I noticed that the game was fetched multiple times.
I assume that `loadGame` is a callback listening to `profile` and during this process, the limits are checked, so the profile is refetched, so the function change, and so the game is loaded again.
It's not mandatory but it saves unnecessary calls

Reply:
Perfect then :)